<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSelectDeviceDialog</class>
 <widget class="QDialog" name="QSelectDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Device</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="layoutBody">
   <item>
    <widget class="QLabel" name="lblDeviceList">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border: 2px solid rgb(91,130,181);
	background-color : rgb(227,234,242);
	color : rgb(44,66,97);
	font-weight: bold;
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>Device List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblDeviceList">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
	background-color: white;
	border: 2px solid rgb(91,130,181);
	selection-background-color: rgb(91,130,181);
	border-radius: 5px;
	padding: 1px;
}</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>No.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Model Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Device Path</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBtnBox">
     <item>
      <widget class="QPushButton" name="btnPlayBuzzer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 2px solid #8f8f91;
	border-radius: 6px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);
	min-width: 80px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
	border: none;
}

QPushButton:focus {
	outline: none;
}

QPushButton:default {
	border-color: rgb(90, 129, 129);
}</string>
       </property>
       <property name="text">
        <string>Play Buzzer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerBtn">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 2px solid #8f8f91;
	border-radius: 6px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);
	min-width: 80px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
	border: none;
}

QPushButton:focus {
	outline: none;
}

QPushButton:default {
	border-color: rgb(90, 129, 129);
}</string>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
